pkgs/desktops/deepin/deepin-icon-theme/default.nix

```diff
@@ -1,28 +1,35 @@
-{ stdenv, fetchFromGitHub, gtk3, papirus-icon-theme, deepin }:
+{ stdenv, fetchFromGitHub, gtk3, xcursorgen, papirus-icon-theme, deepin }:
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "deepin-icon-theme";
-  version = "15.12.64";
+  version = "15.12.68";
 
   src = fetchFromGitHub {
     owner = "linuxdeepin";
     repo = pname;
     rev = version;
-    sha256 = "0z1yrp6yg2hb67azrbd9ac743jjh83vxdf2j0mmv2lfpd4fqw8qc";
+    sha256 = "12jgz81s5qggmnkfg9m5f799r10p43qmh4zqxl1kjvlrqgvsc9rf";
   };
 
-  nativeBuildInputs = [ gtk3 papirus-icon-theme ];
+  nativeBuildInputs = [ gtk3 xcursorgen ];
+
+  buildInputs = [ papirus-icon-theme ];
 
   postPatch = ''
     patchShebangs .
 
-    # install in $out
-    sed -i -e "s|/usr|$out|g" Makefile tools/hicolor.links
-
     # keep icon-theme.cache
     sed -i -e 's|\(-rm -f .*/icon-theme.cache\)|# \1|g' Makefile
   '';
+  
+  buildTargets = "all hicolor-links";
+  installTargets = "install-icons install-cursors";
+  installFlags = [ "PREFIX=$(out)" ];
```

Comment:
You should (when you can) use `placeholder ''out''` for this.

Reply:
Done.

Reply:
Still seeing these other references
https://gist.github.com/worldofpeace/6ea1e7f41573916d9c64cb036031e971

Reply:
Done.

Reply:
Thanks :+1: